seqr/views/utils/individual_utils.py

```diff
@@ -125,7 +125,13 @@ def _update_from_record(record, user, families_by_id, individual_lookup, updated
         record['displayName'] = ''
 
     # Update the parent ids last, so if they are referencing updated individuals they will check for the correct ID
-    if record.get('maternalId') or record.get('paternalId'):
+    if 'father' in record or 'mother' in record:
+        parent_updates.append({
```

Comment:
Are any tests covering the branches for having 'father' or 'mother' attributes?

Reply:
yes, thats the case that happens when maternal and paternal guid are provided in the request body which has a new test: https://github.com/broadinstitute/seqr/pull/3247/files#diff-5b097568e0ddce955255883bc1931cfc244741bc6dfa15e1820e86b7b06f4561R163